src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/components/_alerts.scss

```diff
@@ -1,78 +1,26 @@
 .alert {
-  border: 0;
-  line-height: 40px;
-  margin-top: 65px;
-  position: fixed;
-  width: 100%;
-  z-index: 99;
+  padding: 15px;
```

Comment:
Dit kan ingesteld worden in $alert-padding in _bootstrap-variables.scss, regel 631